README.md

```diff
@@ -1 +1,63 @@
-# curriculum-html-css
\ No newline at end of file
+# Setup-and-Mobile-First
+
+Main page of the personal portfolio is implemented
+
+## Built With
+
+- HTML and CSS
+- Linter
+
+## Getting Started
+
+- First of all set up the .github folder, then inside it create a folder named as workflows, then inside that foloder create a yml file called linters.yml. Then take the corresponding github file for the HTML/CSS linter config.
+- Then add index.html and a style.css to style our main page.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- node.js
+
+### Setup
+
+Through Github
+
+### Install
+
+Through Github
+
+### Usage
+
+- Personal Portfolio
+
+### Run tests
+
+- Through Github
+
+### Deployment
+
+- Not deployed yet
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/mkaya13)
+- Twitter: [@twitterhandle](https://twitter.com/mkaya133)
+- LinkedIn: [LinkedIn](https://twitter.com/mkaya133)
+
+## 🤝 Contributing
+
+Thanks the microverse team their great contributions.
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Inspired by the microverse option 1 template.
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This link is broken, kindly fix it by adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯